### ISSUE TITLE
v8.0.0 install fails because of missing postinstall script

### ISSUE THREAD

**Issue by amiuhle** [erikras/redux-form#4291]
### Are you submitting a **bug report** or a **feature request**?

**bug report**

### What is the current behavior?

`v8.0.0` does not install because `scripts/patch-immutable-flow-error.js` is not part of `files`.

### Other information

```bash
node_modules/redux-form: Command failed.
Exit code: 1
Command: node scripts/patch-immutable-flow-error.js && opencollective-postinstall

Error: Cannot find module '/path/to/project/node_modules/redux-form/scripts/patch-immutable-flow-error.js
```

**Comment by amiuhle** [comment]
Maybe it's not even needed, previously there were two `postinstall` properties in `package.json`. The second one probably overwrote the first one:

https://github.com/erikras/redux-form/commit/e46a2b8566d1bfce4416d00668517c8d929daa15

**Comment by amiuhle** [comment]
Fixed in `v8.0.1`, thanks!
